beshop/js/script.js

```diff
@@ -1,136 +0,0 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-undef */
-(function($) {
-  document.addEventListener('DOMContentLoaded', function () {
-    $('.sidenav').sidenav();
-    $('.modal').modal();
-    $('select').formSelect();
-
-    let breakpoint = window.matchMedia('(min-width: 768px)');
-
-    let homepageSlider = function () {
-      if (breakpoint.matches) {
-        let desktopSlider = new Swiper('.sales_header', {
-          simulateTouch: false,
-          slidesPerView: 'auto',
-          loop: true,
-          effect: 'coverflow',
-          centeredSlides: true,
-          centeredSlidesBounds: true,
-          coverflowEffect: {
-            rotate: 0,
-            stretch: 0,
-            depth: 70,
-            modifier: 1,
-            slideShadows: false
-          },
-          pagination: {
-            el: '.swiper-pagination'
-          },
-          navigation: {
-            nextEl: '.swiper-button-next',
-            prevEl: '.swiper-button-prev'
-          }
-        });
-      }
-      else {
-        let mobileSlider = new Swiper('.sales_header', {
-          simulateTouch: false,
-          effect: 'slide',
-          loop: true,
-          slidesPerView: '1',
-          pagination: {
-            el: '.swiper-pagination'
-          },
-          navigation: {
-            nextEl: '.swiper-button-next',
-            prevEl: '.swiper-button-prev'
-          }
-        });
-      }
-    };
-
-    homepageSlider();
-    breakpoint.addListener(homepageSlider);
-
-    function desktopSearch() {
-      let searchButton = document.querySelector('.open-search');
-      let searchWidget = document.querySelector('.widget_search');
-      let searchInput = document.querySelector('.widget_search .search-field');
-
-      searchButton.onclick = function(event) {
-        event.stopPropagation();
-        searchWidget.classList.toggle('active');
-        searchInput.focus();
-      }
-
-      document.addEventListener('click', function(event) {
-        if (searchWidget.classList.contains('active')) {
-          let isClickInside = searchInput.contains(event.target);
-
-          if (!isClickInside) {
-            searchWidget.classList.toggle('active');
-          }
-        }
-      });
-    }
-
-    desktopSearch();
```

Comment:
do we have this anywhere else?

Reply:
accidentally cleared the file. gonna restore